tests/test_cli.py

```diff
@@ -149,7 +149,7 @@ def test_validate_command_usage():
 
 
 def test_validate_command_with_edx_format():
-    """Test the validate command using the edx format."""
+    """Tests the validate command using the edx format."""
```

Comment:
Not sure we are homogeneous across the whole project...

Reply:
Good catch) Thanks) Yes, indeed)

Reply:
Created a separate pull request for this part)